scripts/start-multiple-alerts-scan.gmp

```diff
@@ -138,13 +147,12 @@ def get_alerts(gmp, sender_email, recipient_email, debug=False):
     alert_id = None
     alert = alert_object.xpath('alert')
     if len(alert) == 0:
-        print("creating alert")
```

Comment:
Removing this `print` statement is a good idea to keep the output clean, but could you please take a look at the other `print` statements in this script as well to keep the output consistent?